docs/editor/accessibility.md

```diff
@@ -25,12 +25,76 @@ You can adjust the Zoom level in VS Code with the **View** > **Appearance** > **
 
 When you adjust the zoom level with the **View** > **Zoom In / Out** commands, the zoom level is persisted in the `window.zoomLevel` [setting](/docs/getstarted/settings.md). The default value is 0 and each increment/decrement changes the zoom level by 20%.
 
+## Customizing warning colors
```

Comment:
Let's move this below "Color vision accessibility" since it is more specific that selecting an existing theme.
So the flow would be:
High Contract theme
Color vision accessiblity (with "selecting accessible colors" as a nested H3)
Customizing warning colors

Reply:
@gregvanl makes sense!